pypeit/telescopes.py

```diff
@@ -90,11 +90,19 @@ def __init__(self):
 class GeminiSTelescopePar(TelescopePar):
     def __init__(self):
         super(GeminiSTelescopePar, self).__init__(name='GEMINI-S',
-                                              longitude=70.8062,              # Longitude of the telescope (NOTE: West should correspond to positive longitudes)
-                                              latitude=-30.1691,              # Latitude of the telescope
-                                              elevation=2200.0,               # Elevation of the telescope (in m)
+                                              longitude=70.7367,              # Longitude of the telescope (NOTE: West should correspond to positive longitudes)
+                                              latitude=-30.24075,              # Latitude of the telescope
+                                              elevation=2750.0,               # Elevation of the telescope (in m)
                                               diameter=8.1)
 
+class SOARTelescopePar(TelescopePar):
```

Comment:
If you can find an eff_aperture please add that since it is better for throughput estimates. Sometimes these numbers are hard to find, but try to find it if you can.

Reply:
found central obscuration numbers but they didn't make a ton of sense..
http://www.ctio.noao.edu/soar/content/soar-technical-specs

looks like 4.1m is correct to ~5%